src/NzbDrone.Core/Extras/Others/OtherExtraService.cs

```diff
@@ -15,71 +15,41 @@ namespace NzbDrone.Core.Extras.Others
 
         public override int Order => 2;
 
-        public override IEnumerable<ExtraFile> CreateAfterSeriesScan(Series series, List<EpisodeFile> episodeFiles)
-        {
-            return Enumerable.Empty<ExtraFile>();
-        }
-
-        public override IEnumerable<ExtraFile> CreateAfterEpisodeImport(Series series, EpisodeFile episodeFile)
+        public override IEnumerable<ExtraFile> CreateAfterMovieScan(Movie movie, List<MovieFile> movieFiles)
         {
             return Enumerable.Empty<ExtraFile>();
         }
 
-        public override IEnumerable<ExtraFile> CreateAfterEpisodeImport(Series series, string seriesFolder, string seasonFolder)
+        public override IEnumerable<ExtraFile> CreateAfterMovieImport(Movie movie, MovieFile movieFile)
         {
             return Enumerable.Empty<ExtraFile>();
         }
 
-        public override IEnumerable<ExtraFile> MoveFilesAfterRename(Series series, List<EpisodeFile> episodeFiles)
-        {
-            // TODO: Remove
-            // We don't want to move files after rename yet.
-
-            return Enumerable.Empty<ExtraFile>();
-
-            var extraFiles = _otherExtraFileService.GetFilesBySeries(series.Id);
+        public override IEnumerable<ExtraFile> MoveFilesAfterRename(Movie movie, List<MovieFile> movieFiles)
+        { 
+            var extraFiles = _otherExtraFileService.GetFilesByMovie(movie.Id);
             var movedFiles = new List<OtherExtraFile>();
 
-            foreach (var episodeFile in episodeFiles)
+            foreach (var movieFile in movieFiles)
             {
-                var extraFilesForEpisodeFile = extraFiles.Where(m => m.EpisodeFileId == episodeFile.Id).ToList();
+                var extraFilesForMovieFile = extraFiles.Where(m => m.MovieFileId == movieFile.Id).ToList();
 
-                foreach (var extraFile in extraFilesForEpisodeFile)
+                foreach (var extraFile in extraFilesForMovieFile)
                 {
-                    var existingFileName = Path.Combine(series.Path, extraFile.RelativePath);
-                    var extension = Path.GetExtension(existingFileName).TrimStart('.');
-                    var newFileName = Path.ChangeExtension(Path.Combine(series.Path, episodeFile.RelativePath), extension);
-
-                    if (newFileName.PathNotEquals(existingFileName))
-                    {
-                        try
-                        {
-                            _diskProvider.MoveFile(existingFileName, newFileName);
-                            extraFile.RelativePath = series.Path.GetRelativePath(newFileName);
-                            movedFiles.Add(extraFile);
-                        }
-                        catch (Exception ex)
-                        {
-                            _logger.Warn(ex, "Unable to move extra file: {0}", existingFileName);
-                        }
-                    }
+                    movedFiles.AddIfNotNull(MoveFile(movie, movieFile, extraFile));
```

Comment:
Sorry should have been more specific with the renaming code. I meant this bit here. Sonarr seems to rename and move the metadata file.

Reply:
This bit of code was just not eliminated, but moved up to the base class `ExtraFileManager` to eliminate duplication same as upstream.